packages/back-end/types/organization.d.ts

```diff
@@ -241,6 +241,17 @@ export type OrganizationMessage = {
   level: "info" | "danger" | "warning";
 };
 
+export type GetStartedChecklistItem = {
+  step: string;
+  isCompleted: boolean;
+};
+
+export type GetStartedChecklist = {
+  features: GetStartedChecklistItem[];
+  experiments: GetStartedChecklistItem[];
+  importedExperiments: GetStartedChecklistItem[];
+};
```

Comment:
Breaking this up by list isn't going to work well.  There are shared steps like setting up environments.  Better to just have a single array.

Reply:
Also, it's much simpler if we can just do an array of strings instead of an array of objects.  This will let us do really efficient Mongo queries to add/remove a string from the set without needing to worry about default states, merging, or race conditions.